match/docs/openapi/orchestrator/find.yaml

```
@@ -3,7 +3,7 @@ post:
   tags:
     - "Match"
   summary: "Search for all matching participant records using de-identified data"
-  description: "Searches all state databases for any participant records that are an exact match to the `lds_hash` of persons provided in the request."
+  description: "Searches all state(except initiating state) databases for any participant records that are an exact match to the `lds_hash` of persons provided in the request."
```

```suggestion
  description: "Searches all state databases for any participant records that are an exact match to the `lds_hash` of persons provided in the request. Does not search the database for the state initiating the request."
```
What do you think about this language? Trying to be as clear as possible. We could also defer to @rserey or @cjhskippy.

*Chore:* Once the language is finalized, can you also update the description in `docs/openapi/duplicate-participation-api.yaml` to match and then re-generate docs. The description for the public-facing markdown documentation is stored there. This should result in a change to `docs/openapi/generate/duplicate-participation-api/openapi.md`.

---

@kingcomma  looks good to me.